.github/workflows/ci-cd.yml

```
@@ -187,25 +138,129 @@ jobs:
+      - name: Deploy to FastAPI Production EC2
+        uses: appleboy/ssh-action@v0.1.6
+        with:
+          host: ${{ secrets.AI_EC2_HOST }}          # 새로운 FastAPI EC2
+          username: ${{ secrets.AI_EC2_USER }}      # ec2-user
+          key: ${{ secrets.AI_EC2_SSH_KEY }}        # 새로운 EC2 키
+          port: 22
+          timeout: 600s
+          script: |
+            set -e
+            echo "🚀 FastAPI AI 서비스 배포 시작"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)